### PR TITLE
Add summary41.txt

### LINKED ISSUE #25998: 0.D PR title summarization 41
This is one of a number of issues to crowd-source categorization of PR titles in order to construct a changelog for the 0.D release.

 Please prepend a category title to each line listed at the end of this issue and commit the lines as a new file in the repository with the provided name.

 Feel free to summarize or expand on the titles to make them better fit in the changelog.

 See https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt for previous changelog entry examples.

 # TITLES
 The categories to choose from are: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N
 See the Changelog Guidelines at https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md for explanations of the categories.

 # FILENAME 
 summary41

 # Lines to summarize

Reset selected mission before follower training
Changes long stick deconstruction recipe to not require cutting tool + other related changes
[DONE] Crop locust spawn fix
Add hint for toggle base trait in wish_mutate menu
Rag bundles
Breaking long sticks as an item action
Remove irremovable mods after the gun is destroyed by fire
remove junk from plain popcorn 
Fix placement of basement stairs
Handle UTF-8 string when format safemode message
Handle using consoles with no map data
Added an electric clay kiln, portable and vehicle.
chickenfeed: make chickens and other small domestic birds tameable
signoff: add mlangsdorf's signoff
submenu for controlling multiple vehicle electronics
vehicles: Open all doors
Ice lab temperature and food decay fix nr 2
Mortar & pestle migration to mainline (vanilla) from More Survival Tools Mod 
Quick fix: Breaking stick removal method changed to consume_items
Jsonify mil. surplus
Jsonify radio station
The Trencher
Don't show ammo for bows
Improve handling of butchered flesh
New callbacks for Lua
Reload will ignore bows in vicinity
Acorn meal decay time fix and recipe change
Capitalize scenario name
Describe vehicle parts Part 1a
Added a chance for small utility/construction vehicles to spawn on construction sites
Fix localization of computers
Fix mycus fruit
Include computer interface strings in translation template
Fix food causing game to crash at chargen
[DONE] Player Faction Base
[RDY] Fix type conversion mistake in healing_rate_medicine function
First draft of json-augmentation of labs
Artifact dreams
Added chicken cages.
item.cpp: remove incorrect double negation for pistol crossbow check
Jsonify sport shop
Fixes bandage effect not working on torso or head
Easy item repairs should prevent rust
Replace instant healing with slow healing effects.
[READY] Added engine blocks and engine deconstruction recipes.
Added missing NPC name parameter to add_msg call in npc::set_attitude().
[DONE] Crafting Cache Fix
[DONE] Debug Menu Rename NPCs
[DONE] Faction Camp Upgrade Vehicle Check
[DONE] Remove/recover camp manager
[DONE] Trench and OM Sight Check
Horde improvements: Better displays & zombie lurkers
Make one makeshift bandage at a time
Leadworks guns rework
Rain fills all
Digging and filling pits/mounds rework
Jsonify pharmacy
Moved manual CBM installation to Bright Nights mod
Update map cache before spawning monsters
Another fix for bandage effect.
Fix incorrect volume of ethanol in crafting recepies.
When backing out of options menu, don't handle resizing screen if terminal size values haven't changed
Corrected dispersion of few  rivtech shotguns
Jsonized part of ranged balance
Show error message on wielding popup
Display vehicle descriptions
empty vehiclepart descriptions fix
more vehicle controls QoL changes
Fixed examining rubble invoked DIG instead of a CLEAR_RUBBLE
Adjust Henry Big Boy  description to be less wrong.
Adds Toolbox to Wire Cutting list. Try #2.
Allow return of reference to monster in Lua
Stop item dupe
[DONE] Blacksmith Shop Faction Base Expansion
Update Atomic Nightlight Vehicle Description
Making RELOAD_AND_SHOOT weapons correctly drop steadiness after shot
Allow adding to existing monster groups in json
Make shaving/haircut interruptable
blazemod: add magazines and increase clip capacities
Lab json improvements
More Complete and Rebalanced Vitamins
Rescaled skill dependent ranged dispersion
Fix deconstruction failure message
Fix for garlic bulb split consuming all garlic bulbs.
Automatically balance hardcoded & json lab & lab finales against each…
Arrow/Bolt changes
Bow/crossbow rebalance
Anesthesia spawns in stacks of 3
Safe autodoc mod, a dependency of Bright Nights
Moves stillsuit's effects to json flags
Fix inventory letter assignment getting cleared after trade
Gluten and Lactose free alternative recipes.
[DONE] Simple Camp Bugs
Spawn dirt below the dumpsters of the fast food restaurant.
Bugfix: finale shocker/autodoc weight 25, not 2500.
Improve place_nested: Fuzzy neighbor matching and an else option
Move worn climate control to a JSON flag
New Bow Attachments
Fix for ranged weapon dispersion UI doesn't do math
Leadworks underslung shotgun now has masterkey stats

### PR DESCRIPTION
**Pull request by tomkuba**
SUMMARY: None

Annotates changelog entries for summary 41.
Fixes #25998

### CODE REVIEW

File: summary41.txt [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Interface Add hint for toggle base trait in wish_mutate menu
+Content Rag bundles
+Features Breaking long sticks as an item action
+Bugfixes Remove irremovable mods after the gun is destroyed by fire
+Balance Remove junk from plain popcorn
+Bugfixes Fix placement of basement stairs
+Infrastructure Handle UTF-8 string when format safemode message
+Infrastructure Handle using consoles with no map data
+Content Added an electric clay kiln, portable and vehicle
+Features Make chickens and other small domestic birds tameable (Chickenfeed)
```
Comment (Leland) [inline summary41.txt]: `tamable`

File: summary41.txt [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Features Horde improvements: Better displays & zombie lurkers
+Content Add makeshift bandages to improvised first aid kit
+Balance Leadworks guns rework
+Features Rain fills all
+Features Digging and filling pits/mounds rework
+Content Jsonify pharmacy
+Mods Manual CBM installation moved to Bright Nights mod
+Bugfixes Update map cache before spawning monsters
+Bugfixes Fix for bandage effect
+Bugfixes Fix incorrect volume of ethanol in crafting recepies
```
Comment (Leland) [inline summary41.txt]: recipes

File: summary41.txt [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Bugfixes Fixed examining rubble invoked DIG instead of a CLEAR_RUBBLE
+Bugfixes Adjust Henry Big Boy description to be less wrong
+Features Add Toolbox to Wire Cutting list
+Mods Allow return of reference to monster in Lua
+Bugfixes Stop item dupe
+Features Blacksmith Shop Faction Base Expansion
+Balance Update Atomic Nightlight Vehicle Description
+Bugfixes Making RELOAD_AND_SHOOT weapons correctly drop steadiness after shot
+Mods Allow adding to existing monster groups in json
+Balance Make shaving/haircut interruptable
```
Comment (Leland) [inline summary41.txt]: `interruptible`

File: summary41.txt (hunk 1 above)
Comment (tomkuba) [reply summary41.txt]: Ups, stop the review please. I'll fix my spell checker and update PR.

Comment (Leland) [reply summary41.txt]: No worries 😄 these 3 were the only ones I found.

You might want to change uses of "json" to all caps as well while you're at it btw

**Pull request merged**